api.bs

```diff
@@ -527,10 +560,12 @@ The arguments to <a method for=PrivateAttribution>saveImpression()</a> are as fo
 
 Given <a dictionary lt=PrivateAttributionImpressionOptions>|options|</a>:
 
-1.  Collect the implicit API inputs:
-    1. The current timestamp
-    1. The impression site domain
-    1. The iframe site domain
+1.  Collect the implicit API inputs from the [=relevant settings object=]:
+    1.  The timestamp is set to the [=current high resolution time=].
+    2.  The [=impression site=] is set to the [=site=] from the [=top-level origin=].
+    3.  The [=intermediary site=] is set to the [=site=] from [=origin=]
+        if that is not [=same site=] with the [=top-level origin=].
```

Comment:
We can reference https://html.spec.whatwg.org/multipage/browsers.html#obtain-a-site for these.